### PR TITLE
Feature/template ruby

### PR DESCRIPTION
**Pull request by antoniofilhozup**
# Pull Request

## What I did

Ruby formula template 

## How I did it
I created new files according to the structure of the existing languages.

## How to verify it

Just run `rit create formula` and select `ruby` as language

## What kind of change does this PR make

- [ ] Major
- [x] Minor
- [ ] Patch

## Description for the changelog
Ruby formula template

### CODE REVIEW

File: templates/create_formula/languages/ruby/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,13 @@
+FROM ruby:2.6
+
+USER root
+
```
Comment (fabianofernandeszup) [inline templates/create_formula/languages/ruby/Dockerfile:4]: Add docker instalation 
```
RUN curl -fsSL https://commons-repo.ritchiecli.io/install.sh | bash
```

File: templates/create_formula/languages/ruby/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,13 @@
+FROM ruby:2.6
```
Comment (joaorufinozup) [inline templates/create_formula/languages/ruby/Dockerfile:1]: Please, use the same docker image in this file and in the config.json file. In config.json file, you are using "dockerImageBuilder": "cimg/ruby:2.7.1"

File: templates/create_formula/languages/ruby/Dockerfile (hunk 1 above)
Comment (maurineimirandazup) [reply templates/create_formula/languages/ruby/Dockerfile:4]: Done.

File: templates/create_formula/languages/ruby/Dockerfile (hunk 2 above)
Comment (maurineimirandazup) [reply templates/create_formula/languages/ruby/Dockerfile:1]: Done.

**Pull request merged**